app/client/src/redux/ducks/authDuck.js

```diff
@@ -0,0 +1,117 @@
+/* 
```

Comment:
Why are we including the Apache license when we've got the apache license file at the root of the project?

Reply:
Those were there in all files when we started the project.

Reply:
I was thinking that too. It was in every code file in the project when we started so we assumed it had to be there.